constraints.txt

```diff
@@ -10,7 +10,8 @@ onnx==1.16.0
 onnxruntime==1.17.1
 
 # TensorFlow
-tensorflow==2.12.1
+tensorflow==2.12.1; python_version < '3.9'
+tensorflow==2.15.1; python_version >= '3.9'
```

Comment:
Why we can not have only one version of TF here?

Reply:
It is required because we still support Python 3.8 and run tests using it. It is possible to drop this, but in that case, we should end Python 3.8 support.